test/fw/ptl/lib/ptl_resourceresv.py

```diff
@@ -503,12 +503,6 @@ def receive_alarm(signum, stack):
         script_path = self.du.create_temp_file(hostname=hostname,
                                                body=script_body,
                                                suffix='.py')
-        if not self.du.is_localhost(hostname):
```

Comment:
Why did you remove this?

Reply:
It's checking to see if it's not localhost, and if it's not it copies the file out there.  Then a few lines lower ("if not self.du.chmod.."), it does a chmod on that hostname/script_path again and that chmod does a copy again by running run_cmd.  This should reduce the copy calls and increase the speed of some of the tests.

Reply:
From the code, it looks like create_temp_file() does a remote copy as well if hostname is not localhost. So, ya, this seems unnecessary, thanks for removing it.